vic/drivers/shared_all/src/put_data.c

```diff
@@ -735,8 +753,6 @@ collect_wb_terms(cell_data_struct cell,
             tmp_cond2 = param.HUGE_RESIST;
         }
         out_data[OUT_AERO_COND2].data[0] += tmp_cond2;
-    }
-    if (overstory) {
```

Comment:
There were two `if (overstory)` statements in a row.  I just removed the extra one.